packages/components/workbench/stories/Workbench.stories.tsx

```diff
@@ -42,7 +42,7 @@ export default {
 
 interface Args {
   ['Header Title']: string;
-  ['Header Description']: string;
+  ['Header Description']: string | ReactElement;
```

Comment:
nit: `ReactNode` has `ReactElement | string | number`  by default

Reply:
I know,I kept as this since this is how this is used on the title on the component too, probably because we don't want numbers only? 🤔 
but I can change it anyway, WDYT??

Reply:
I'd say that for description it doesn't make much sense, even a number should be allowed if someone needs it

Reply:
yeah I don't see a reason to prevent numbers as well

Reply:
Yeah and actually the users can even hack it, doing something like "123" even for the title 😅